src/server/service/scenarios/db.js

```diff
@@ -0,0 +1,53 @@
+const { sql } = require('../../util/sqlHelpers');
+const { withClient, withClientTransaction } = require('../../util/db');
+
+exports.getScenario = async function getScenario(scenarioId) {
+    return withClient(async client => {
+        const results = await client.query(
+            sql` SELECT * FROM scenario WHERE id = ${scenarioId};`
+        );
+        return results.rows[0];
+    });
+};
+
+exports.addScenario = async function addScenario(authorId, title, description) {
+    return withClientTransaction(async client => {
```

Comment:
no need to transaction a single statement AFAIK

Reply:
I think you're right! From this doc:

>PostgreSQL actually treats every SQL statement as being executed within a transaction. If you do not issue a BEGIN command, then each individual statement has an implicit BEGIN and (if successful) COMMIT wrapped around it. A group of statements surrounded by BEGIN and COMMIT is sometimes called a transaction block.

https://www.postgresql.org/docs/9.1/tutorial-transactions.html